server/src/database/model-helpers.ts

```
@@ -0,0 +1,54 @@
+import _ from 'lodash';
+import { Context } from '../config/context';
+
+/**
+ * autoUpdatedTimeFields contains timestamp fields that are automatically
+ * updated by the database.
+ */
+const autoUpdatedTimeFields = ['createdAt', 'lastModified'] as const;
+
+export const convertToBackendModel =
+  <D extends Object | null | undefined>() =>
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)